stanford_media.module

```diff
@@ -279,17 +280,40 @@ function stanford_media_media_prepare_form(MediaInterface $media, $operation, Fo
   // Display a message to the user to alert them than editing will affect
   // multiple pieces of content.
   if ($count) {
-    $message = \Drupal::translation()->formatPlural(
+    $plural_text = \Drupal::translation()->formatPlural(
       $count,
-      'Changing this media item will affect %count piece of content.',
-      'Changing this media item will affect %count pieces of content.',
-      ['%count' => $count]
+      '@count piece of content',
+      '@count pieces of content',
+      [
+        '@count' => $count,
+      ]
+    );
+    $formatted_link = new FormattableMarkup(
+      '<a href="@usage" target="_blank">@plural_text</a>',
+      [
+        '@usage' => $media->toUrl('usage')->toString(),
+        '@plural_text' => $plural_text,
+      ]
+    );
```

Comment:
`\Drupal\Core\Link::fromTextAndUrl()` does this for you.


```suggestion
    $formatted_link = Link::fromTextAndUrl($plural_text, $media->toUrl('usage'));
```

Reply:
Ok, fixed.